scripts/build_server_docker

```diff
@@ -4,5 +4,27 @@ set -o errexit
 set -o xtrace
 
 readonly SCRIPTS_DIR="$(dirname "$0")"
+readonly OAK_REMOTE_CACHE_KEY='./.oak_remote_cache_key.json'
+
+echo "TEST"
+# Do we have a json key for the remote cache
```

Comment:
(nit) JSON

Reply:
Also worth adding a link to https://docs.bazel.build/versions/master/remote-caching.html#google-cloud-storage around here.

Reply:
Done, both